webpackShims/ui-bootstrap.js

```diff
@@ -1,11 +1,27 @@
 define(function (require) {
```

Comment:
This file is specifically for the `ui-bootstrap` dependency. I think it would be better if we moved the new logic into `src/ui/public/chrome/api/translations.js`